lib/licenses.nix

```diff
@@ -1139,6 +1139,13 @@ lib.mapAttrs mkLicense ({
     fullName = "Sendmail License";
   };
 
+  sfl = {
+    fullName = "Source First License 1.1";
+    url = "https://gitlab.futo.org/videostreaming/grayjay/-/blob/master/LICENSE.md";
```

Comment:
Is there no more canonical URL?  If not, maybe would make more sense to just inline this since it seems unlikely to be used for anything else.

Reply:
I suggested that this be added to licenses.nix; given what FUTO seems to have planned, I would be surprised if this remains their only SFL-licensed program suitable for inclusion in NixOS.

Reply:
Okay, that's fine.